Packs/Malwarebytes/TestPlaybooks/Malwarebytes-Test.yml

```diff
@@ -0,0 +1,1169 @@
+id: Malwarebytes-Test
+version: 6
+name: Malwarebytes-Test
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: add37d10-7ce8-44bc-8759-20cf466b4dae
+    type: start
+    task:
+      id: add37d10-7ce8-44bc-8759-20cf466b4dae
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "1":
+    id: "1"
+    taskid: 9efae663-09b0-4cc7-8ea5-95274e858b2b
+    type: regular
+    task:
+      id: 9efae663-09b0-4cc7-8ea5-95274e858b2b
+      version: -1
+      name: DeleteContext
+      script: DeleteContext
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "2"
+    scriptarguments:
+      all:
+        simple: "yes"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 200
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "2":
+    id: "2"
+    taskid: 1bfd868f-bb3b-4676-8ad7-487564b049f2
+    type: regular
+    task:
+      id: 1bfd868f-bb3b-4676-8ad7-487564b049f2
+      version: -1
+      name: malwarebytes-scan-and-remediate
+      script: '|||malwarebytes-scan-and-remediate'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      hostname:
+        simple: ${inputs.Hostname}
```

Comment:
you are relying on hostname to be provided as input, so please add it to the playbook inputs